getting-started/templates/systemlink-values.yaml

```diff
@@ -1011,6 +1011,11 @@ argoworkflows:
       ## Maximum number of workflows that can run in parallel.
       ##
       parallelism: &workflowParallelism 300
+      ## Uncomment to specify instanceID for Argo Workflows. This is needed to avoid conlicts with already existing
+      ## Argo Workflows deployments in the cluster.
+      # instanceID:
+      #   enabled: true
+      #   explicitID: sl-notebook-execution-0
```

Comment:
Is the name `sl-notebook-execution-0` something we defined and therefore something the user must use, or is this an example where the actual value is something the user must provide?

Reply:
We don't define by default, this just an example what the customer needs to do in case they have another argo workflows deployed.

Reply:
Makes sense. Leaving as it, but fixing a small spelling mistake.